vm/src/builtins/type.rs

```diff
@@ -403,15 +403,14 @@ impl PyType {
         zelf.iter_mro().map(|cls| cls.clone().into()).collect()
     }
 
-    #[pymethod(name = "__ror__")]
     #[pymethod(magic)]
-    pub fn or(zelf: PyObjectRef, other: PyObjectRef, vm: &VirtualMachine) -> PyObjectRef {
-        if !union_::is_unionable(zelf.clone(), vm) || !union_::is_unionable(other.clone(), vm) {
-            return vm.ctx.not_implemented();
-        }
+    pub fn ror(zelf: PyObjectRef, other: PyObjectRef, vm: &VirtualMachine) -> PyObjectRef {
+        _or(other, zelf, vm)
+    }
```

Comment:
when ror and or share same implementation, please keep the single function share both method like original code.

Reply:
If or, and ror in type.rs were placed in a same function, the validation of union's order couldn't be guaranteed.
I've already tested and checked the test was failed where they're combined.
That's why I make them just like that.

Reply:
Oh, you are right! I didn't catch that point.